Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewModel.swift

```diff
@@ -47,40 +45,13 @@ class DefaultBrowserOnboardingViewModel {
     //  Internal vars
     var model: DefaultBrowserOnboardingModel?
     var goToSettings: (() -> Void)?
-    private let experiments: NimbusApi
-    
-    private(set) lazy var displayTitleImage = !(experiments.withVariables(featureId: .onboardingDefaultBrowser).getBool("should-hide-title-image") ?? false)
 
-    init(experiments: NimbusApi = Experiments.shared) {
-        self.experiments = experiments
+    init() {
         setupUpdateModel()
     }
-
-    private func getCorrectImage() -> UIImage {
-        let layoutDirection = UIApplication.shared.userInterfaceLayoutDirection
-        switch ThemeManager.instance.currentName {
-        case .dark:
-            if layoutDirection == .leftToRight {
-                return UIImage(named: "Dark-LTR")!
-            } else {
-                return UIImage(named: "Dark-RTL")!
-            }
-        case .normal:
-            if layoutDirection == .leftToRight {
-                return UIImage(named: "Light-LTR")!
-            } else {
-                return UIImage(named: "Light-RTL")!
-            }
-        }
-    }
```

Comment:
Since we no longer need that "Light-LTR" we can remove the image files as well

<img width="1303" alt="Screen Shot 2021-09-14 at 12 34 54 PM" src="https://user-images.githubusercontent.com/8919439/133297920-c276ecf0-c1f1-4dba-8970-f1c767b98a3a.png">